podcasts/TranscriptsViewController.swift

```diff
@@ -541,6 +542,9 @@ extension TranscriptsViewController: TranscriptSearchAccessoryViewDelegate {
     }
 
     private func processMatch() {
+        guard !searchIndicesResult.isEmpty, currentSearchIndex >= 0, currentSearchIndex < searchIndicesResult.count else {
+            return
+        }
```

Comment:
I don't believe we need al those checks, we can do it like this:


```suggestion
        if searchIndicesResult.isEmpty {
            return
        }

```

Also not using a `guard` negation — we've been avoid doing those.